build-locally.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+#This script build the monolithic image locally, for cases like the raspberry pi, which operates on arm64 architecture, which isn't officially supported.
+
+PURPLEBOLD="$(tput setf 5 bold)"
+
+printf "${PURPLEBOLD}Building temporary modified Ubuntu image:\n"
+docker build -t lancachenet/ubuntu:latest --progress tty https://github.com/lancachenet/ubuntu.git
+
+printf "${PURPLEBOLD}Removing standart Ubuntu image:\n"
+docker rmi ubuntu
```

Comment:
Why would you need to remove a local Ubuntu image which isn't referenced anywhere? You already remove lancachenet/Ubuntu later?

Reply:
It's referenced in lancache-ubuntu's dockerfile, so it gets pulled to make it.

Reply:
Alright, now that you mention it i see it might be a issue for some people. I will see if i can somehow check at the beginning of the script if there's already an ubuntu image present, and not remove it if it is. Any advice on that?